src/internal.c

```diff
@@ -23310,8 +23310,10 @@ int SendFinished(WOLFSSL* ssl)
  * Returns 0 on success
  */
 static int CreateOcspRequest(WOLFSSL* ssl, OcspRequest* request,
-                             DecodedCert* cert, byte* certData, word32 length)
+                             DecodedCert* cert, byte* certData, word32 length,
+                             byte *takeOwnership)
```

Comment:
takeOwnership => ctxHasRequest

Why not update pointer in lock protected code?